cognite/client/__init__.py

```diff
@@ -1,3 +1,4 @@
 from cognite.client._cognite_client import CogniteClient
 
+
 __version__ = "2.13.0"
```

Comment:
```suggestion
__version__ = "2.13.1"
```

Reply:
Why not 2.14 btw?

Reply:
My bad, I agree with 2.14
_MINOR version when you add functionality in a backwards compatible manner_

Reply:
updated